src/main/java/ru/skypro/homework/dto/Role.java

```diff
@@ -1,5 +1,13 @@
 package ru.skypro.homework.dto;
 
-public enum Role {
-    USER, ADMIN
+import org.springframework.security.core.GrantedAuthority;
+
+public enum Role implements GrantedAuthority {
+    USER, ADMIN;
+
+    @Override
+    public String getAuthority() {
+        return name();
```

Comment:
authority начинается с ROLE_, поэтому здесь надо вернуть
```suggestion
        return "ROLE_" + name();
```